src/engine/node/mod.rs

```diff
@@ -378,6 +378,8 @@ impl Node {
         *self.m().color()
     }
 
-
+    pub fn children(&self) -> &Vec<Node> {
+        &(*self).children
```

Comment:
you're right, I typed something wrong and it didn't compile so I did this to make the compiler happy, but `&self.children` actually works